src/components/Reports/MPGAIncomeExpensesReport/MPGAIncomeExpensesReport.tsx

```diff
@@ -63,7 +67,32 @@ export const MPGAIncomeExpensesReport: React.FC<
     },
   });
 
-  const { incomeData, expenseData } = useFilteredFunds(reportData);
+  const transformedData: Funds[] = (
```

Comment:
You could make this a little more terse by spreading the fields with the `...` operator that aren't transformed.

Also, this should probably be memoized.